### ISSUE TITLE
clustertools documentation needs proper integration (Trac #725)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#719]
Migrated from https://code.icecube.wisc.edu/ticket/725
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:13:59", 
    "description": "", 
    "reporter": "nega", 
    "cc": "blaufuss, nwhitehorn, olivas", 
    "resolution": "fixed", 
    "_ts": "1458335639172163", 
    "component": "documentation", 
    "summary": "clustertools documentation needs proper integration", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2014-04-15T18:29:35", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20141003T17:54:07: 
* nega changed cc from "blaufuss nwhitehorn" to "blaufuss, nwhitehorn, olivas"
* nega changed summary from "clustertools needs to be added to the big boy documentation" to "clustertools documentation needs proper integration"

**Comment by icecube-trac** [comment]
Trac update at 20150519T21:10:17: 
* david.schultz changed status from "new" to "accepted"
* david.schultz changed milestone from "" to "offline V15-07-00"

**Comment by icecube-trac** [comment]
Trac update at 20150709T21:59:30: david.schultz changed milestone from "offline V15-07-00" to "cvmfs py2-v2"

**Comment by icecube-trac** [comment]
Trac update at 20150720T16:38:55: 
* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 135340:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="135340"
> add cvmfs documentation. not sure this is the best place, but it puts it near the top of the page, which is desired. fixes #719
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas changed milestone from "cvmfs py2-v2" to ""
* olivas commented:

> Milestone cvmfs py2-v2 deleted
